examples/tv-casting-app/android/App/app/src/main/java/com/chip/casting/app/ConnectionFragment.java

```diff
@@ -120,15 +121,57 @@ private void beginCommissioning(
       FailureCallback onConnectionFailure,
       SuccessCallback<ContentApp> onNewOrUpdatedEndpoints) {
     Log.d(TAG, "Running commissioning");
+    Object commissioningComplete =
```

Comment:
Nit to match the other callback objects

```suggestion
    Object commissioningCompleteCallback =
```

Reply:
Also why is this just `Object`?

Reply:
Done and switched to specific type.